arbos/l1pricing/l1pricing.go

```diff
@@ -126,9 +131,11 @@ func OpenL1PricingState(sto *storage.Storage) *L1PricingState {
 		sto.OpenStorageBackedUint64(unitsSinceOffset),
 		sto.OpenStorageBackedBigUint(pricePerUnitOffset),
 		sto.OpenStorageBackedBigInt(lastSurplusOffset),
+		sto.OpenStorageBackedBigInt(calldataPriceOffset),
```

Comment:
Informational: I thought to suggest only doing it in arbos50 - but then realised it's o.k. to open it in older arbos as well, and makes upgrade simpler.